Formula/cmusfm.rb

```diff
@@ -0,0 +1,66 @@
+class Cmusfm < Formula
+  desc "Last.fm standalone scrobbler for the cmus music player"
+  homepage "https://github.com/Arkq/cmusfm"
+  url "https://github.com/Arkq/cmusfm/archive/v0.3.3.tar.gz"
+  sha256 "9d9fa7df01c3dd7eecd72656e61494acc3b0111c07ddb18be0ad233110833b63"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --disable-dependency-tracking
+      --disable-silent-rules
+    ]
+
+    system "autoreconf", "--install"
+    mkdir "build" do
+      system "../configure", *args
+      system "make", "install"
+    end
+  end
+
+  test do
+    cmus_home = testpath/".config/cmus"
+    cmusfm_conf = cmus_home/"cmusfm.conf"
+    cmusfm_sock = cmus_home/"cmusfm.socket"
+    cmusfm_cache = cmus_home/"cmusfm.cache"
+
+    cmusfm = bin/"cmusfm"
+    test_artist = "Test Artist"
+    test_title = "Test Title"
+    status_args = %W[
+      artist #{test_artist}
+      title #{test_title}
+      duration 31
+    ]
+
+    mkpath cmus_home
+    touch cmusfm_conf
+
+    begin
+      server = fork do
+        exec cmusfm, "server"
+      end
+      loop do
+        sleep 0.5
+        assert_equal nil, Process.wait(server, Process::WNOHANG)
+        break if cmusfm_sock.exist?
+      end
+
+      system cmusfm, "status", "playing", *status_args
+      sleep 31
```

Comment:
Let me know if you'd prefer faking `time`.

Reply:
Not having a mandatory 31 second sleep seems desirable.